apps/client/src/locales/messages.nl.xlf

```diff
@@ -0,0 +1,2650 @@
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+  <file source-language="en" target-language="nl" datatype="plaintext" original="ng2.template">
+    <body>
+      <trans-unit id="e19fcf996343543e13789f17b550ab0c08124b5c" datatype="html">
+        <source>Create Account</source>
+		<target state="translated">Account aanmaken</target>
```

Comment:
Can you please replace the tabs with two spaces that it matches the style?

Reply:
fixed.